src/test/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/CreateSDORespondent1NotificationHandlerTest.java

```diff
@@ -50,13 +55,17 @@ public class CreateSDORespondent1NotificationHandlerTest extends BaseCallbackHan
     private OrganisationService organisationService;
     @Autowired
     private CreateSDORespondent1NotificationHandler handler;
+    private final String defendantEmail = "respondent@example.com";
+    private final String legacyCaseReference = "create-sdo-respondent-1-notification-000DC001";
+    private final String defendantName = "respondent";
 
     @Nested
     class AboutToSubmitCallback {
 
         @BeforeEach
         void setup() {
             when(notificationsProperties.getSdoOrdered()).thenReturn("template-id");
+            when(notificationsProperties.getSdoOrderedSpecBilingual()).thenReturn("template-id");
```

Comment:
"template-id" could be put into a constant string so you dont have to repeat it everywhere

Reply:
done